app/javascript/components/CreateSurvey.js

```diff
@@ -20,37 +20,64 @@ const questionType = {
 const CreatSurvey = () => {
     const [baseUrl, setBaseUrl] = useState('');
     const [surveyName, setSurveyName] = useState('');
+    const [surveyId, setSurveyId] = useState('');
```

Comment:
I might be confused here, but with the id be held as a string or an int?

Reply:
good catch, Ill fix that to a -1